next.config.js

```diff
@@ -1,9 +1,17 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {
+const withPWA = require("next-pwa")({
+  dest: "public",
+  register: true,
+  skipWaiting: true,
+});
+const nextConfig = withPWA({
+  experimental: {
+    newNextLinkBehavior: true,
+  },
+
   compiler: {
-    styledComponents: false,
+    styledComponents: true,
   },
-  transpilePackages: ["next-auth"],
   reactStrictMode: true,
```

Comment:
Why did you change the styledComponents and next auth settings?

Reply:
Oh hatte die Next config vorbereitet bevor ich next auth integriert hatte 😅, binde ich wieder ein